src/main/kotlin/ru/shiroforbes/web/Router.kt

```diff
@@ -246,7 +246,7 @@ fun Routing.routes(
 
     authenticate("auth-session-admin-only") {
         get("/transactions/exercises/countryside") {
-            val countryside = studentService!!.getGroup(GroupType.Countryside)
+            val countryside = studentService!!.getGroup(GroupType.Countryside).sortedBy { student -> student.firstName() }
```

Comment:
It is better to sort by both names, not just the first/last.

Reply:
done